views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+                      </tr>
+                      <tr>
+                        <th>twitter</th>
+                        <td>twitter handle: <code>example</code> (for <code>@example</code>)</td>
+                      </tr>
+                      <tr>
+                        <th>facebook</th>
+                        <td>Facebook name: <code>AnneExample</code> (for <code>facebook.com/AnneExample)</code>
+                      </tr>
+                      <tr>
+                        <th>group</th>
+                        <td>name of political group <code>Example Faction</code></td>
+                      </tr>
+                      <tr>
+                        <th>group_id<span class="superscript">&dagger;</span></th>
+                        <td>id for group</td>
+                      </tr>
+                      <tr>
+                        <th>area</th>
+                        <td>name of area represented <code>Example County</code></td>
+                      </tr>
+                      <tr>
+                        <th>area_id<span class="superscript">&dagger;</span></th>
+                        <td>id for area</td>
+                      </tr>
+                      <tr>
+                        <th>chamber<span class="superscript">&dagger;</span></th>
+                        <td>name of chamber <code>House of Representatives</code></td>
+                      </tr>
+                      <tr>
+                        <th>term<span class="superscript">&dagger;</span></th>
+                        <td>id or index for term</td>
+                      </tr>
+                      <tr>
+                        <th>start_date</th>
+                        <td>start of membership (if needed) <code>2010-11-29</code></td>
+                      </tr>
+                      <tr>
+                        <th>end_date</th>
+                        <td>end of membership (if needed) <code>2012-07-14</code></td>
+                      </tr>
+                      <tr>
+                        <th>image</th>
+                        <td>URL to image file <code>http://example.com/example.jpg</code></td>
+                      </tr>
+                      <tr>
+                        <th>gender</th>
+                        <td>gender if known <code>M</code> or <code>F, or <code>male</code> or <code>female</code></td>
+                      </tr>
+                    </table>
+                  </div>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p>
+          <span class="superscript">&dagger;</span>
+          Values of fields marked &dagger; will be consistent throughout the
+          legislature: if their values are equal (string comparison), then
+          they’re the same thing. So all politicians with the same
+          <code>area_id</code>, for example, are representing the same area.
```

Comment:
This is a slightly confusing description. It seems like it's trying to counter some other assumption, but without stating what that is. It's unclear how two `start_date`s that are the same string value refer to two different things, for example, or even two `area` fields.